### DISCUSSION TITLE
can not download

### DISCUSSION THREAD

**Discussion by megadroid3** [termux/termux-app#2797] [Q&A] [answered]
Cannot download for htc android 6

**Comment by ChiefMikeK** [comment]
Android 6 support has been terminated but legacy packages can be found here
- https://archive.org/details/termux-repositories-legacy 
- A Newer apk is under development so stay tuned for more details

**Comment by sylirre** [comment, marked as answer]
https://github.com/termux/termux-app/wiki/Termux-on-android-5-or-6
